pandas/core/indexes/multi.py

```diff
@@ -324,11 +324,17 @@ def from_arrays(cls, arrays, sortorder=None, names=None):
                    codes=[[0, 0, 1, 1], [1, 0, 1, 0]],
                    names=['number', 'color'])
         """
+        error_msg = "Input must be a list / sequence of array-likes."
```

Comment:
We could maybe think about how to improve the actual message as well, because on a first read I was interpreting this as "Input must be [a list] or [a sequence of array-likes]" (while of course it is "[list or sequence] of array-likes"), which confused me at first ..

To be true to the code, what it actually needs to be is a "list-like of list-likes"? Which is also not that nice to write .. 
I am wondering if a more strict error message (stricter than what we allow), something like "Input must be a list of arrays" is not actually easier to understand for users.

Reply:
I was also thinking that the messages should be changed from `Input must be...` to something along the lines of `'arrays' parameter of MultiIndex.from_arrays must be...` and then regurgitate whatever is in the docstring.

Reply:
> I am wondering if a more strict error message (stricter than what we allow), something like "Input must be a list of arrays" is not actually easier to understand for users.

IIUC the reason that a sequence is accepted is to provide backward compatibility with zip. So sequence does not necessarily need to be mentioned in the docstring.